README.md

```diff
@@ -10,4 +10,396 @@ TODO: description
 
 ## API Routes
 
-TODO: document API routes
+### Shared Account
+
+#### `POST /api/accounts` - Create an new shared account
+
+**Body**
+
+- `name` _{string}_ - The account owner's name
+- `username` _{string}_ - The user's username
+- `password` _{string}_ - The user's password
+
+**Returns**
+
+- A success message
+- The created shared account's details
+
+**Throws**
+
+- `403` if there is a user already logged in
+- `400` if `name` is empty or missing
+- `400` if `username` or `password` is not in correct format or missing
+- `409` if `username` is already in use
+
+#### `PATCH /api/accounts` - Update a shared account's details
+
+**Body**
+
+- `name` _{string}_ - The new name
+
+**Returns**
+
+- A success message
+- The updated shared account details
+
+**Throws**
+
+- `401` if the user is not logged in
+- `400` if `name` is empty
+
+#### `DELETE /api/accounts` - Delete shared account
+
+**Returns**
+
+- A success message
+
+**Throws**
+
+- `401` if the user is not logged in
+
+#### `POST /api/accounts/credentials` - Create an new credential for a shared account
```

Comment:
Should we also have a route to `get` all credentials for an account? So a user could see all authorized parties at once?

Reply:
I thought of this while writing as well. What I ended up realizing is that when you log in you get the "account details" which can be both the account name and list of associated usernames which can then just be stored on the front end. What do you think of this?

Reply:
I like that! Sounds good to me.